knodle/labeler/CheXpert/utils.py

```diff
@@ -0,0 +1,47 @@
+"""
+Define helper functions.
+"""
+import os
+import pandas as pd
+import numpy as np
+
+from .config import CheXpertConfig
+
+
+def t_matrix_fct(config: CheXpertConfig) -> pd.DataFrame:
+    """Create T-matrix from rules (mentions)."""
+
+    mentions = pd.concat([pd.read_csv(os.path.join(config.mention_data_dir, file), header=None).assign(
+        classes=os.path.basename(file).split('.')[0]) for file in config.files], ignore_index=True)
+
+    T_matrix = pd.DataFrame(data=mentions).iloc[:, 1].str.get_dummies().to_numpy()
+
```

Comment:
I'm not a particular fan of these complex pandas constructs (both lines). I know a lot, maybe even most people do it, but I think it's unreadable and definitly needs testing